extra_foam/offline/file_server.py

```diff
@@ -225,8 +225,10 @@ def run(self):
             fast_devices = [("*DET/*CH0:xtdf", "image.data")]
         elif detector == "JungFrau":
             fast_devices = [("*/DET/*:daqOutput", "data.adc")]
-        elif detector == "FastCCD":
+        elif detector in ["FastCCD"]:
```

Comment:
`== "FastCCD"`

Reply:
It is all 'in' in the new commit. I hope the detector people will not invent more naming convention.